js/foam/u2/DAOController.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * @license
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+CLASS({
+  package: 'foam.u2',
+  name: 'DAOController',
+  extends: 'foam.u2.View',
+  documentation: 'Expects a DAO as $$DOC{ref:".data"}. Renders a ' +
+      '$$DOC{ref:".listView"} of $$DOC{ref:".rowView", plural:true} for it. ' +
+      'Supports editing and creating new views through the ' +
+      '$$DOC{ref:"foam.u2.DAOUpdateController"} and ' +
+      '$$DOC{ref:"foam.u2.DAOCreateController"}.',
+
+  requires: [
+    'foam.u2.DAOUpdateController',
+    'foam.u2.DAOListView',
+  ],
+
+  imports: [
+    'stack',
+  ],
+  exports: [
+    'data as dao',
```

Comment:
Yes. `DAOCreateController` expects to import it, and it's not the only place. It seems to me to make sense for DAO views to export that DAO as `dao`.